module/Motion/CorrectingRotation.cpp

```diff
@@ -0,0 +1,153 @@
+/**
+ * @file   CorrectingRotation.h
+ * @brief  角度補正回頭
+ * @author aridome222
+ */
+
+#include "CorrectingRotation.h"
+
+using namespace std;
+
+CorrectingRotation::CorrectingRotation(int _targetAngle, double _targetSpeed)
+  : targetAngle(_targetAngle), targetSpeed(_targetSpeed)
+{
+}
```

Comment:
```suggestion
{};
```
セミコロン抜け。いらないと思いますが、他ファイルはあるので一応。
書いておいてなんですが、なくても Google Test, build ともに通るので大丈夫だとは思います。